### ISSUE TITLE
Bug | Editing email template

### ISSUE THREAD

**Issue by TumbaoJu** [akretion/overdue-reminder#5]
Hello Alexis,
We love your module and are using it but we discovered that once the module installed, if you modify the email template (just click on "Edit" button), and save, the body of the email does'nt appears correctly.

![Sélection_999(164) (1)](https://user-images.githubusercontent.com/38360781/121027518-055cc900-c775-11eb-94fb-1588f9392bc0.png)

Looking at the email template after the edition, I can see that the "% for" parts are moved outside the table.
![Sélection_999(543)](https://user-images.githubusercontent.com/38360781/121028183-97fd6800-c775-11eb-877f-466f88295354.png)

We discovered that the bug comes from the Summernote Edior.
![pasted image 0](https://user-images.githubusercontent.com/38360781/121028511-e6126b80-c775-11eb-8740-9b1ac67243f7.png)

The anomaly can be reproduced on a standard Summernote editor.
The widget automatically extracts the conditioning instruction out of the table.

Did you already notice that problem?  Do you have a solution in mind?

We tried adding a personalised report (using aeroo) but the module delete any report we could add to the email template.

Thank you in advance for you help.

**Comment by mgielissen** [comment]
This is the problem

`if object.counter > 2:`

**Comment by TumbaoJu** [comment]
@alexis-via Hello Alexis! Hope you are doing well.  Have you seen this issue?  Do you have an answer for me?
Thank you.

**Comment by alexis-via** [comment]
Yes, I have already noticed this issue. I don't have a real solution so far ; the workaround I use it to edit the HTML code instead of using the WYSIWYG editor.

**Comment by TumbaoJu** [comment]
@alexis-via Merci! ;)
